pmm-app/tests/verifyQAN_test.js

```diff
@@ -226,3 +226,64 @@ xScenario('PMM-T123 - Verify User is able to search for filter value', async (I,
+xScenario('PMM-T100 Check Changing Main Metric, PMM-T203 Verify user is able to search for columns by typing, PMM-T133, PMM-T132', async (I, qanPage, dashboardPage) => {
+    const metricName = 'Query Count with errors';
+    const urlString = 'num_queries_with_errors';
+    I.waitForElement(qanPage.fields.newQANAddColumn, 30);
+    I.waitForElement(qanPage.tableHeaderColumnLocator("Load"), 30);
+    I.click(qanPage.tableHeaderColumnLocator("Load"));
+    I.waitForElement(qanPage.fields.newQANMetricDropDown, 30);
+    I.fillField(qanPage.fields.newQANColumnSearchField, metricName);
+    I.click(dashboardPage.fields.metricTitle);
+    I.waitForElement(qanPage.tableHeaderColumnLocator(metricName), 30);
+    I.seeElement(qanPage.tableHeaderColumnLocator(metricName));
+    I.dontSeeElement(qanPage.tableHeaderColumnLocator('Load'));
+    I.seeInCurrentUrl(urlString);
+    const url = await I.grabCurrentUrl();
+    I.amOnPage(url);
+    I.waitForElement(qanPage.fields.newQANAddColumn, 30);
+    I.waitForElement(qanPage.tableHeaderColumnLocator(metricName), 30);
+    I.dontSeeElement(qanPage.tableHeaderColumnLocator("Load"));
+});
+
+// TODO: Uncomment after new QAN will be merged
+xScenario('PMM-T99 Verify User is able to add new metric, PMM-T222 Verify `Add column` dropdown works', async (I, qanPage, dashboardPage) => {
+    const metricName = 'Query Count with errors';
+    const urlString = 'num_queries_with_errors';
+    I.waitForElement(qanPage.fields.newQANAddColumn, 30);
+    I.waitForElement(qanPage.tableHeaderColumnLocator("Load"), 30);
+    I.click(qanPage.fields.newQANAddColumn);
+    I.waitForElement(qanPage.fields.newQANMetricDropDown, 30);
+    I.fillField(qanPage.fields.newQANColumnSearchField, metricName);
+    I.click(dashboardPage.fields.metricTitle);
+    I.waitForElement(qanPage.tableHeaderColumnLocator(metricName), 30);
+    I.seeElement(qanPage.tableHeaderColumnLocator(metricName));
+    I.seeElement(qanPage.tableHeaderColumnLocator('Load'));
+    I.seeInCurrentUrl(urlString);
+    const url = await I.grabCurrentUrl();
+    I.amOnPage(url);
+    I.waitForElement(qanPage.fields.newQANAddColumn, 30);
+    I.waitForElement(qanPage.tableHeaderColumnLocator(metricName), 30);
+    I.seeElement(qanPage.tableHeaderColumnLocator("Load"));
+    I.seeElement(qanPage.tableHeaderColumnLocator(metricName));
+});
+
+// TODO: Uncomment after new QAN will be merged
+xScenario('PMM-T13 - Verify QAN has MongoDB, MySQL, PostgreSQl all three Service Types', async (I, qanPage) => {
+    const filters = ['mongodb', 'mysql', 'postgres'];
+    qanPage.waitForNewQANPageLoaded();
+    I.waitForElement(qanPage.fields.filterBy, 30);
+    const countBefore = await qanPage.getCountOfItems();
+    for (i = 0; i < filters.length; i++) {
+        await I.fillField(qanPage.fields.filterBy, filters[i]);
+        qanPage.applyFilterNewQAN(filters[i]);
+        I.waitForInvisible(qanPage.fields.newQANSpinnerLocator, 30);
```

Comment:
I wonder, how can we wait for invisible? Is the naming right?

Reply:
yes, I think just waiting for the loader to load data and become invisible, it could be changed to wait for table data also.